### PR TITLE
References Tests: Broken Site Reporting

### PR DESCRIPTION
**Pull request by kdzwinel**
**Reviewer:** Sam

## Description:
Reference tests for broken site reporting.

I also made some tweaks to test running setup.

## Steps to test this PR:
Mostly just run tests, but I also touched the reporting form in the popup, so please make sure that it still works as expected.

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -5,19 +5,19 @@ module.exports = function (grunt) {
     grunt.loadNpmTasks('grunt-execute')
     grunt.loadNpmTasks('grunt-karma')
 
-    let browser = grunt.option('browser')
-    let buildType = grunt.option('type')
+    const browser = grunt.option('browser')
```
Comment (kdzwinel) [inline Gruntfile.js:8]: some simple linting fixes in this file to stop visual studio code from shouting at me

File: karma.conf.js [hunk 2]
```diff
@@ -7,6 +7,9 @@ module.exports = function (config) {
         singleRun: true,
         files: [],
         logLevel: config.LOG_ERROR,
+        browserConsoleLogOptions: {
+            level: 'warn'
+        },
```
Comment (kdzwinel) [inline karma.conf.js:12]: reducing output of tests to warnings only - there was too much trash being logged to the terminal

File: package.json [hunk 3]
```diff
@@ -104,8 +104,7 @@
                 {
                     "exclude": [
                         "transform-regenerator"
-                    ],
-                    "debug": true
```
Comment (kdzwinel) [inline package.json:108]: ditto

**Review by sammacbeth — APPROVED** [review]
All looks good to me.

**Pull request merged**